lib/node-token-validation/src/config/TokenValidationParameters.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { JwtConstants, TokenType } from "../utils/Constants";
+import { JWK } from "jose";
+
+export type TokenValidationParameters = {
```

Comment:
Can we add some comments here detailing the difference between `TokenValidationParameters` and `ValidationParameters`?

Reply:
Yes, added.